cmd/cri-containerd/cri_containerd.go

```diff
@@ -164,6 +164,39 @@ func startProfilingServer(host string, port string) {
 	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 	if err := http.ListenAndServe(endpoint, mux); err != nil {
-		glog.Errorf("Failed to start profiling server: %v", err)
+		logrus.WithError(err).Error("Failed to start profiling server")
+	}
+}
+
+func setLogLevel(l string) error {
+	lvl, err := log.ParseLevel(l)
+	if err != nil {
+		return err
+	}
+	if err := setGLogLevel(lvl); err != nil {
+		return err
+	}
+	logrus.SetLevel(lvl)
+	return nil
+}
+
+// TODO(random-liu): Set glog level in plugin mode.
+func setGLogLevel(l logrus.Level) error {
+	if err := flag.Set("logtostderr", "true"); err != nil {
+		return err
+	}
```

Comment:
alsologtostderr?

Reply:
We don't want to log to files, I think. If we want that someday, we can change to `alsologtostderr`.